src/dialog/src/Dialog.js

```diff
@@ -50,28 +50,88 @@ const animationStyles = {
 
 class Dialog extends React.Component {
   static propTypes = {
+    /**
+     * Composes the Overlay component as the base.
+     */
     ...Overlay.propTypes,
+
+    /**
+     * Children can be a node or a function accepting `({ close })`.
+     * See an example to understand how this works.
+     */
+    children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
+
+    /**
+     * Title of the Dialog. Titles should use Title Case.
+     */
     title: PropTypes.node,
+
+    /**
+     * Props passed through to the primary button.
+     * Passing this object will show the button and cancel button.
+     * You should pass `children` and `onClick`.
+     */
+    primaryButton: PropTypes.object,
```

Comment:
So it can't be boolean?

Reply:
Nope.

Reply:
Most of the time you wouldn't really want to pass props to the primary button though?

Then you'll end up with lots of code like `primaryButton={{}}`.

Reply:
You have to pass the onClick handler

Reply:
And you need to pass in children to give it a label. You always have to define something right?